hydrus/data/resource_based_classes.py

```diff
@@ -34,6 +37,40 @@ def get_type(object_: Dict[str, Any]) -> str:
     return type_
 
 
+def get_modified_object(object_: Dict[str, Any], doc: HydraDoc, path: str) -> Dict[str, Any]:
+    """
+    Return the object after modifying properties.
+    :param object_: Dict containing object properties
+    :param doc: HydraDoc object
+    :param path: Path of the Class or Collection
+    :return: object_ containing properties
+    """
+    temp_object = copy.deepcopy(object_)
```

Comment:
why you need to copy this object?

Reply:
oops ! I was using this in a previous approach. Thanks for catching that.